zebra/zebra_evpn_mh.c

```diff
@@ -3101,6 +3101,9 @@ static void zebra_evpn_es_show_entry(struct vty *vty, struct zebra_evpn_es *es,
 				json_array_string_add(json_flags, "local");
 			if (es->flags & ZEBRA_EVPNES_REMOTE)
 				json_array_string_add(json_flags, "remote");
+			if (es->flags & ZEBRA_EVPNES_LOCAL &&
```

Comment:
I'm a bit concerned about this: the two flags involved are already represented - do we need to capture this semantic in this way, explicitly? can't the operator decide what the presence or absence of the two flags means?